### DISCUSSION TITLE
Snap Package ready and working well

### DISCUSSION THREAD

**Discussion by soumyaDghosh** [flxzt/rnote#654] [Show and tell]
Hello devs of rnote, this app is wonderful, and I wanted to use it as a snap cause I love snaps :smiling_face_with_tear: I have posted the snap version of this app from v0.5.17 and it's working very well. It's currently active in 118+ devices. I would like to know what the devs think about this. Will you guys please feature this in your repo. If you wish, I can pass the ownership of that snap to you, and the updates can be served directly from this repo. 
The manifest repo: https://github.com/soumyaDghosh/rnote-snap
The snapcraft website: https://snapcraft.io/rnote

**Comment by PhilDevProg** [comment]
@flxzt said in an issue that he's personally committed to FlatPak and someone else would have to maintain other packages including a Snap: https://github.com/flxzt/rnote/issues/174#issuecomment-1128756774

**Comment by PhilDevProg** [comment]
The screenshots of the Snap should also be the same as the screenshots of the Flatpak in my opinion.

**Comment by soumyaDghosh** [comment]
The snap doesn't look the same as the flatpak, it has its own unique style, it completely on the user, what he has set as accent color or what is set by the distro in the stylesheet. Saying that, the screenshots are nothing different. I don't think that'll make much of a difference.

**Comment by flxzt** [comment]
I am opposed to actively advertising the app as snap and especially the snap store (given that it is proprietary and a vehicle for canonical to lock parts of the ecosystem to their own products). I can't stop you from packaging it as snap, but I don't want to have to support it nor do I want to advertise it.

If you do decide that you want to keep maintaining it: Please add a disclaimer to the snap saying that it is unofficial and not supported by upstream. Thanks!

**Comment by soumyaDghosh** [comment]


**Reply by Kneemund** [reply]
> SNAP STORE IS NOT A PROPRIETARY PROJECT

But... did you read the content of the pages that you've linked? 🤔
The snap store is neither visible nor open source, and is explicitly mentioned as being "proprietary" - exactly as flxzt said. Frontend software like `snapd` and Ubuntu's app store being open source doesn't change that.

![image](https://user-images.githubusercontent.com/49598842/235314232-11282759-90a1-4ffc-8a71-f0a3558df1a8.png)

> And regarding mentioning it unofficial, I haven't mentioned it as official either. 

Okay, but that doesn't change anything about the request to be explicit about it and mentioning the resulting lack of support (from us).

**Reply by soumyaDghosh** [reply]
I am sorry, I am wrong about that then.

**Comment by soumyaDghosh** [comment]
Hii, kindly mention the snap package as an unofficial package atleast.

```
<a href="https://snapcraft.io/rnote">
  <img alt="Get it from the Snap Store" src="https://snapcraft.io/static/images/badges/en/snap-store-black.svg" />
</a>
```

**Comment by soumyaDghosh** [comment]
@flxzt The Unofficial Snap Package currently has more active users than the official counterparts. Do you still want to keep it as unofficial? I will be there to help you manage the snap. But, atleast show it in the store that it's published by you?

![image](https://github.com/flxzt/rnote/assets/72045785/1a4dc9ef-ece0-4a68-bf4c-aade6f2cc2cd)

Currently it has **691 ACTIVE** users
